src/Components/Header.jsx

```diff
@@ -0,0 +1,44 @@
+import { useContext } from "react";
+import { Link } from "react-router-dom";
+import PostContext from "./PostContext";
+
+function Header() {
+  const { loggedInUser, dataLoading, userLoading, setFocusUser } =
+    useContext(PostContext);
+  const handleProfileClick = () => {
+    setFocusUser(loggedInUser);
+  };
+
+  return (
+    <header id="page-header">
+      <svg
```

Comment:
SVGs are best placed as separate components to avoid makingthe code hard to read